dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/utils.py

```diff
@@ -231,9 +231,8 @@ def find_value_in_map(a_map, the_key):
 #
 # @return : the key associated to the value, if the value exists, None otherwise
 def find_key_in_map(a_map, the_value):
-    for key, value in a_map.items():
-        if value == the_value:
-            return key
+    if the_key in a_map:
```

Comment:
"the_key" doesn't seem to be defined. Is it normal?

Reply:
Yes, I badly reported it into the script

Reply:
Yes = you are correct this is wrong :) it is solved now.